portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,13 +37,17 @@
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<>();
+    // Integer guaranteed in script.js
```

Comment:
Couldn't access the link but I saw you set `limit` to a string. Originally I thought using a drop-down list would have been safer than a number input, but apparently there are ways to get around that too ... Now I know that every user input should be assessed on the back end. 

`limit` can't be negative integers either. So that should throw an error as well.

Reply:
Done.